gprofiler/client.py

```diff
@@ -113,7 +121,14 @@ def get_health(self):
         return self.get("health_check")
 
     def submit_profile(
-        self, start_time: datetime.datetime, end_time: datetime.datetime, profile: str, total_samples: int
+        self,
+        start_time: datetime.datetime,
+        end_time: datetime.datetime,
+        profile: str,
+        total_samples: int,
+        spawn_time: float,
+        cpu_avg: Optional[float],
```

Comment:
Encapsulate in some "metrics" object (so later we can add more fields with less frtiction)